pkg/scheduler/api/podgroup_info/job_info.go

```diff
@@ -76,6 +77,10 @@ type PodGroupInfo struct {
 	StalenessInfo
 
 	schedulingConstraintsSignature common_info.SchedulingConstraintsSignature
+
+	// inner cache
```

Comment:
PodStatusIndex & ActiveAllocatedCount are also types of inner cache

Reply:
Good point - I will move them into this section as well